src/ReleaseHistory.md

```diff
@@ -390,4 +390,5 @@
 * BUGFIX: `threadFlowLocation.kind` value is getting lost during pre-release transformation. https://github.com/microsoft/sarif-sdk/issues/1502
 * BUGFIX: `location.logicalLocation` convenience setter mishandles null. https://github.com/microsoft/sarif-sdk/issues/1514
 * BUGFIX: Upgrade schemas to latest version (remove `draft-04` from `$schema` property and change `id` to `$id`). This is necessary because the schemas use the `uri-reference` format, which was not defined in draft-04. https://github.com/microsoft/sarif-sdk/issues/1521
-* API BREAKING: The `Init` methods in the Autogenerated SARIF object model classes are now `protected virtual`. This enables derived classes to add additional properties without having to copy the entire code of the `Init` method.
\ No newline at end of file
+* API BREAKING: The `Init` methods in the Autogenerated SARIF object model classes are now `protected virtual`. This enables derived classes to add additional properties without having to copy the entire code of the `Init` method.
+* BUGFIX: Transformation from SARIF 1.0 to 2.x throws Argument-Out-of-Range exception if `result.locations` is an empty array. https://github.com/microsoft/sarif-sdk/issues/1526
```

Comment:
>Argument-Out-of-Range exceptio [](start = 54, length = 30)

May as well just say the type name `ArgumentOutOfRangeException`.

Reply:
i agree! why didnt i!

---
In reply to: [292205908](https://github.com/microsoft/sarif-sdk/pull/1527#discussion_r292205908) [](ancestors = 292205908)